src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/X86Base.PlatformNotSupported.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    /// <summary>
+    /// This class provides access to the x86 base hardware instructions via intrinsics
+    /// </summary>
+    internal static class X86Base
```

Comment:
@jkotas, would you say this is one that we should take to API review or leave be?

C/C++ expose intrinsics for it and it also has a 16-bit overload. Alternatively, we could propose/review general overloads that take `byte`, `sbyte`, `short`, and `ushort`.

If we aren't going to take these, we should probably leave a remark indicating as such and the reason why (they are covered by the more general use `BitOperations` methods).

Reply:
I do not think these should be public APIs.

Reply:
Same reason as what's discussed in https://github.com/dotnet/runtime/issues/953

Reply:
Thanks!

@saucecontrol, could you update the remarks section to indicate these are not public as they are sufficiently covered by corresponding methods in `BitOperations`?

Reply:
Will do.  Also wasn't sure about whether to follow the abstract class pattern of the others since the class will become public if https://github.com/dotnet/runtime/issues/785 comes to be.  I assume this will have to remain standalone since we can't change the base class of the other x86 ISAs so static class is correct?

Reply:
I'm fairly certain changing the base class is fine, all members are static (and therefore also non-virtual) and so it only impacts what members are visible through IntelliSense.

Reply:
Ah yes... I'll leave it for now.  The decision on changing the base class of the others can be taken if/when `X86Base` is made public